cmd/kubeadm/app/phases/controlplane/manifests.go

```diff
@@ -293,10 +293,10 @@ func getControllerManagerCommand(cfg *kubeadmapi.ClusterConfiguration, k8sVersio
 	// Let the controller-manager allocate Node CIDRs for the Pod network.
 	// Each node will get a subspace of the address CIDR provided with --pod-network-cidr.
 	if cfg.Networking.PodSubnet != "" {
-		maskSize := calcNodeCidrSize(cfg.Networking.PodSubnet)
+		maskSize := calcNodeCidrSize(cfg.Networking.PodSubnet) // TODO(awander): Needs to be fixed once PR #73977 lands
 		defaultArguments["allocate-node-cidrs"] = "true"
 		defaultArguments["cluster-cidr"] = cfg.Networking.PodSubnet
-		defaultArguments["node-cidr-mask-size"] = maskSize
+		defaultArguments["node-cidr-mask-size"] = maskSize // TODO(awander): Should be a list of maskSizes
```

Comment:
I don't remember exactly, but you should check this, don't know if the first PR will have a limitation regarding this topic